src/test/ui/issues/issue-56685.stderr

```diff
@@ -0,0 +1,40 @@
+error: unused variable: `x`
+  --> $DIR/issue-56685.rs:14:14
+   |
+LL |         E::A(x) | E::B(x) => {}
+   |              ^
```

Comment:
Could we update the primary span to also point at all the individual bindings?

Reply:
Fixed.